tests/activemq5-unit-tests/src/test/java/org/apache/activemq/transport/failover/FailoverConsumerOutstandingCommitTest.java

```diff
@@ -177,6 +177,7 @@ public void run() {
             targetMethod = "processCommitTransactionOnePhase",
             targetLocation = "ENTRY",
             action = "org.apache.activemq.transport.failover.FailoverConsumerOutstandingCommitTest.stopServerInTransaction();return null")})
+   @Ignore("This test is using Byteman. It's not really accurate.")
```

Comment:
If its not really accurate, and is now being permanently disabled, I would just delete it, it can be found in history if needed.

Reply:
It is not so easy searching from history. I would keep it for now. We could remove all ignored tests before the next major release.

Reply:
Searching from history is fairly trivial. Though based on its prior state and this fix I dont really see anyone actually searching for it.

Reply:
Fair enough, I'll drop them